cmd/query-lb-logs/main.go

```diff
@@ -243,10 +247,22 @@ func CheckEnv(serviceAthena *athena.Athena, logger, infoLogger *log.Logger, v *v
 
 	if !isTableFound {
 		logger.Println("Table not found, creating table....")
-		err := createLogTable(serviceAthena, logger, infoLogger, dbName, logBucket)
+		err = createLogTable(serviceAthena, logger, infoLogger, dbName, logBucket)
```

Comment:
This seems to be changing the value of `err` outside the scope. I am not sure you wanted to do that. Maybe a better setup here is to be specific with the error name so you don't trigger the shadow variables linter. 

```suggestion
		errCreateLogTable := createLogTable(serviceAthena, logger, infoLogger, dbName, logBucket)
```